Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 ruby '2.2.2' # this should match `.ruby-version` and docs/setup.md
 
+gem 'about_yml'
```

Comment:
good question. I was following https://github.com/18F/about_yml/ README which mentions adding to the project Gemfile, but in retrospect, it really doesn't make much sense to have it as part of the app since it was most useful on my local system only, for bootstrapping the file.